.vs/SafeNeighbourhood/config/applicationhost.config

```diff
@@ -157,7 +157,7 @@
       </site>
       <site name="SafeNeighbourhood" id="2">
         <application path="/" applicationPool="SafeNeighbourhood AppPool">
-          <virtualDirectory path="/" physicalPath="C:\Users\rosha\source\repos\Clean_Neighbourhoods\SafeNeighbourhood" />
+          <virtualDirectory path="/" physicalPath="C:\Users\Nikita\Documents\GitHub\Clean_Neighbourhoods\SafeNeighbourhood" />
```

Comment:
Since there is a fully qualified path here, this should likely reside in a gitignore file... or change the path to be relative.